core/src/main/targetconfigs/vz/vz-application.yaml

```diff
@@ -42,74 +40,73 @@ metadata:
   namespace: {{{namespace}}}
 spec:
   workload:
-    apiVersion: weblogic.oracle/v8
-    kind: Domain
-    metadata:
-      name: {{{domainPrefix}}}-domain
-      namespace: {{{namespace}}}
+    apiVersion: oam.verrazzano.io/v1alpha1
+    kind: VerrazzanoWebLogicWorkload
     spec:
-      domainUID: {{{domainUid}}}
+      template:
+        metadata:
+          name: {{{domainPrefix}}}-domain
+          namespace: {{{namespace}}}
+        spec:
+          domainUID: {{{domainUid}}}
 
-      # WebLogic Image Tool provides domainHome, domainHomeSourceType, and imageName
-      domainHome: {{{domainHome}}}
-      domainHomeSourceType: {{{domainHomeSourceType}}}
-      image: {{{imageName}}}
+          # WebLogic Image Tool provides domainHome, domainHomeSourceType, and imageName
+          domainHome: {{{domainHome}}}
+          domainHomeSourceType: {{{domainHomeSourceType}}}
+          image: {{{imageName}}}
 
-      imagePullSecrets:
-        - name: {{{domainPrefix}}}-registry-credentials
-      logHomeEnabled: true
-      includeServerOutInPodLog: true
-      webLogicCredentialsSecret:
-        name: {{{webLogicCredentialsSecret}}}
-      configuration:
-        istio:
-          enabled: false
-        introspectorJobActiveDeadlineSeconds: 900
-        model:
-          configMap: {{{domainPrefix}}}-configmap
-          domainType: {{{domainType}}}
+          imagePullSecrets:
```

Comment:
there is no imagePullSecrets under domain.spec.configuration.model.  See kubectl explain domain.spec.configuration.model

Reply:
imagePullSecrets is not being specified under domain.spec.configuration.model.  It is still being specified under spec.domainUID